src/main/java/software/amazon/event/kafkaconnector/EventBridgeSinkConfigValidator.java

```diff
@@ -89,8 +89,9 @@ public static void validate(ConfigValue configValue, EnvVarGetter getenv) {
         }
 
       case AWS_DETAIL_TYPES_MAPPER_CLASS:
```

Comment:
empty case statement?

Reply:
Just two cases sharing the same validation instead of duplicating rows.

Reply:
OK, wasn't aware the case "falls through" to the next one. Yes, please duplicate here because it's 1/ more explicit and 2/ if someone later reorders the code, the behavior remains